apps/condo/domains/common/components/tasks/TaskProgress.tsx

```diff
@@ -132,18 +126,26 @@ const handledCompletedStatesOfTasksIds = []
 /**
  * Polls tasks record for updates and handles its transition to completed status.
  */
-export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task, storage, calculateProgress, onComplete, translations }) => {
-    const { id } = task
+export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task }) => {
+    const { storage, removeStrategy, calculateProgress, onComplete, translations } = task
+    const { record, stopPolling } = storage.useTask(task.record.id)
 
-    const { record, stopPolling } = storage.useTask(id)
-    const { deleteTask } = useContext(TasksContext)
+    // @ts-ignore
```

Comment:
Please remove as much this lines as possible. I think there is no reason to disable typescript check here